### ISSUE TITLE
Screengrab improperly handles the adb server being out of date.

### ISSUE THREAD

**Issue by EricKuck** [fastlane/fastlane#6285]
### New Issue Checklist

- [X] Updated fastlane to the latest version
- [X] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

### Issue Description
Screengrab thinks "adb" is a valid device name if the adb server is out of date and must be restarted upon running the `adb devices -l` command because it isn't in the array used for `devices.reject`.

##### Complete output when running fastlane, including the stack trace and command used

```
Command run: 
screengrab

Output:
....lots of unrelated stuff.....

$ adb devices -l
▸ List of devices attached
▸ adb server is out of date.  killing...
▸ * daemon started successfully *
▸ emulator-5554          device product:sdk_phone_x86_64 model:Android_SDK_built_for_x86_64 device:generic_x86_64
Multiple connected devices, selecting the first one
To specify which connected device to use, use the -s (specific_device) config option
$ adb -s adb shell echo \$EXTERNAL_STORAGE
▸ error: device 'adb' not found
```

### Configuration Files

Please copy the complete content of your `Fastfile` and any other configuration files you use below:

Not relevant to this issue.

### Environment

fastlane version (run `fastlane -v`):
fastlane 1.104.0

Do you use bundler to execute fastlane (i.e. `bundle exec fastlane`)?
No

Do you use a Ruby environment manager (e.g. `chruby`, `rbenv`, `rvm`)?
No

**Comment by TKBurner** [comment]
@EricKuck Thanks for the commit. Can you create a PR around this so we can fix this? Thanks!

**Comment by EricKuck** [comment]
There is a PR already - #6286

**Comment by mfurtak** [comment]
#6286 has shipped as part of [_screengrab_ v0.5.3](https://github.com/fastlane/fastlane/releases/tag/screengrab%2F0.5.3)
